components/proxy/src/test/java/com/hotels/styx/StyxConfigValidationKtTest.java

```diff
@@ -0,0 +1,99 @@
+/*
+  Copyright (C) 2013-2019 Expedia Inc.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+ */
+package com.hotels.styx;
+
+import com.hotels.styx.config.schema.SchemaValidationException;
+import com.hotels.styx.support.matchers.LoggingTestSupport;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import static ch.qos.logback.classic.Level.ERROR;
+import static ch.qos.logback.classic.Level.INFO;
+import static com.hotels.styx.common.testing.ExceptionExpectation.expect;
+import static com.hotels.styx.support.matchers.LoggingEventMatcher.loggingEvent;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+
+public class StyxConfigValidationKtTest {
```

Comment:
`StyxConfigValidationKtTest`
`____________________^^____`

`Kt` as in Kotlin? In a Java file?

Please implement this in Kotlin. You will get an exception matchers for free, without having to add yet another class. Check our existing Kotlin tests for examples.